cms/templates/js/course-instructor-details.underscore

```diff
@@ -0,0 +1,42 @@
+  <div class="field text field-course-instructor-name">
+  	<label for="course-instructor-name-<%- index %>"><%- gettext("Name") %></label>
+  	<input type="text" class="long" id="course-instructor-name-<%- index %>" value="<%- data['name'] %>" data-index=<%- index %> data-field="name" placeholder="<%- gettext('Instructor Name') %>"  />
+  	<span class="tip tip-stacked"><%- gettext("Please add the instructor's name")%></span>
+  </div>
+
+  <div class="field text field-course-instructor-title">
+    <label for="course-instructor-title-<%- index %>"><%- gettext("Title") %></label>
+    <input type="text" class="long" id="course-instructor-title-<%- index %>" value="<%- data['title'] %>" data-index=<%- index %> data-field="title" placeholder="<%- gettext('Instructor Title') %>" />
+    <span class="tip tip-stacked"><%- gettext("Please add the instructor's title")%></span>
+  </div>
+
+  <div class="field text field-course-instructor-organization">
+    <label for="course-instructor-organization-<%- index %>"><%- gettext("Organization") %></label>
+    <input type="text" class="long" id="course-instructor-organization-<%- index %>" value = "<%- data['organization'] %>" data-index=<%- index %> data-field="organization" placeholder="<%- gettext('Organization Name') %>" />
+    <span class="tip tip-stacked"><%- gettext("Please add the institute where the instructor is associated")%></span>
+  </div>
+
+  <div class="field text field-course-instructor-bio">
+    <label for="course-instructor-bio-<%- index %>"><%- gettext("Bio") %></label>
+    <textarea class="short text" id="course-instructor-bio-<%- index %>" data-index=<%- index %> data-field="bio" placeholder="<%- gettext('Instructor Bio Details') %>" ><%- data['bio'] %></textarea>
+    <span class="tip tip-stacked"><%- gettext("Please add the instructor's bio details")%></span>
+  </div>
+
+  <div class="field image field-course-instructor-image current-instructor-image">
+  <label for="course-instructor-image-<%- index %>"><%- gettext("Image") %></label>
+     <span class="wrapper-instructor-image">
+        <img class="instructor-image" id="instructor-image" src="<%- data['image']%>" alt="<%- gettext('Instructor Image') %>" />
+    </span>
+
+    <input type="text" dir="ltr" class="long new-instructor-image-url" id="course-instructor-image-<%- index %>" value="<%- data['image'] %>" data-field="image" placeholder="<%- gettext('Instructor Image URL') %>" autocomplete="off" />
+    <button type="button" class="action action-upload-instructor-image" data-index=<%- index %>><%- gettext("Upload Image") %></button>
+    <span class="tip tip-stacked"><%- gettext("Please add a photo of the instructor (Note: only JPEG or PNG format supported)")%></span>
+  </div>
+
+  <% if (instructors > 1 ) { %>
+  <div class="actions">
+  	<a href="#" class="button delete-button standard remove-item remove-instructor-data" data-index=<%- index %>><span class="delete-icon"></span><%- gettext("Delete") %></a>
```

Comment:
Instead of using an anchor element, use a button. Getting an anchor element to behave properly for accessibility is very difficult, and buttons will just work.